tests/components/recorder/test_util.py

```diff
@@ -575,9 +576,12 @@ async def test_write_lock_db(hass, tmp_path):
     instance = hass.data[DATA_INSTANCE]
 
     def _drop_table():
+        # This will deadlock in tests with the MutexPool
```

Comment:
Comment added to explain why the test fails, to be removed before the PR is merged